### PR TITLE
Active the test in the CDI extension

### PR DESCRIPTION
**Pull request by otaviojava**
This PR enables the CDI extension test once it was ignored.
Furthermore, it fixes the tests that were broken.

### CODE REVIEW

**Review by fh-ms — APPROVED** [review]

**Comment by otaviojava** [comment]
Hello everyone, is there any news?
Can we move this PR ahead?

I have more contributions to do :)

**Comment by hg-ms** [comment]
> Hello everyone, is there any news? Can we move this PR ahead?
> 
> I have more contributions to do :)

As it has been approved by the reviewer please feel free to Squash and merge

**Comment by otaviojava** [comment]
Sorry, I don't follow you.

But if you don't mind. I'll create a PR for each topic, so I'll wait until this one is merged.

**Pull request merged**
